draft-ietf-httpbis-priority.md

```diff
@@ -205,6 +205,17 @@ Where both endpoints disable HTTP/2 priorities, the client is expected to send
 this scheme's priority signal. Handling of omitted signals is described in
 {{parameters}}.
 
+# Applicability of the Extensible Priority Scheme
+
+The priority scheme defined by this document considers only the prioritization
+of HTTP response data. Clients express priority values to a server, which are
+input into scheduling decisions, see {{client-scheduling}} and
+{{server-scheduling}}.
```

Comment:
```suggestion
The primary use case for the priority scheme defined by this document is the
prioritization of HTTP response data, which is transmitted as HTTP/2 or HTTP/3
DATA frames. Clients express priority values to a server using signals in the
form of a header field {{header-field}} or frame {{frame}}).  Signals are input
into scheduling decisions based on several factors (see {{merging}} and
{{server-scheduling}}). Clients can also use priority values for local
scheduling, see {{client-scheduling}}.
```

Reply:
@kazuho does that address your comments?

Reply:
Thank you for the suggestion. New text is technically correct though I wonder if we can condense the text, or remove references to HTTP/2, HTTP/3, considering the fact that we have been trying to be version-agnostic.

Maybe something as simple as: _The priority scheme defined by this document considers only the prioritization of HTTP messages and the payload of CONNECT method, see {{client-scheduling}}, {{server-scheduling}}, {{CONNECT}}._

Reply:
thanks, I've taken that!